packages/mask/src/plugins/Avatar/Application/NFTListDialog.tsx

```diff
@@ -407,12 +409,10 @@ export function NFTListDialog(props: NFTListDialogProps) {
                     expectedAddress={selectedAccount}>
                     <ChainBoundary
                         expectedChainId={chainId}
-                        predicate={selectedPluginId !== NetworkPluginID.PLUGIN_FLOW ? () => true : undefined}
+                        predicate={supportPluginIds.includes(selectedPluginId) ? () => true : undefined}
                         expectedAccount={selectedAccount}
                         expectedPluginID={
-                            selectedPluginId === NetworkPluginID.PLUGIN_FLOW
-                                ? NetworkPluginID.PLUGIN_EVM
-                                : selectedPluginId
+                            !supportPluginIds.includes(selectedPluginId) ? NetworkPluginID.PLUGIN_EVM : selectedPluginId
```

Comment:
There is no more pluginID except EVM, Solana, and Flow.

Reply:
Yep, But this is in case a new network is added in the future.